core/src/replay_stage.rs

```diff
@@ -850,34 +850,50 @@ impl ReplayStage {
         }
     }
 
+    fn first_of_consecutive_leader_slots(slot: Slot) -> Slot {
+        (slot / NUM_CONSECUTIVE_LEADER_SLOTS) * NUM_CONSECUTIVE_LEADER_SLOTS
+    }
```

Comment:
this might be a good utility function to stick into `leader_schedule_utils.rs`

Reply:
done